src/map/script.c

```diff
@@ -13039,6 +13039,169 @@ static BUILDIN(cloakoffnpc)
 	return true;
 }
 
+
+/*
+ * Create a duplicate of source NPC.
+ * npc_duplicate("<source_npc_name>", "<new_npc_name>", "<new_npc_hidden_name>", "<mapname>", <map_x>, <map_y>, <dir>{, <sprite_id>{, <map_xs>, <map_ys>}});
+ * Return 1 on success, 0 if failed.
+ */
+static BUILDIN(npc_duplicate)
+{
+	int txs = -1, tys = -1;
```

Comment:
Please declare each variable in its own line.

Reply:
done